source/sass/mixins.scss

```diff
@@ -1,3 +1,9 @@
+@mixin heading($font-family, $font-weight, $letter-spacing) {
```

Comment:
This `heading` mixin makes sure all headings will have all three rules set to avoid unintentional styling inheritance.